docs/technical_reference/core/zo_form_libraries.rst

```diff
@@ -0,0 +1,11 @@
+Zero-Order Model Form Libraries
+===============================
+
+WaterTap includes a library of helper methods for assembling common model forms for zero-order type models. This library includes methods for common forms of material balances for different numbers of inlets and outlets, methods for including chemical additions and different approaches for determining electricity consumption. A list of the available methods is given below.
```

Comment:
Are methods for chemical additions a part of this PR too? Or this that for a subsequent PR?

Reply:
That will be the next PR.